.pre-commit-config.yaml

```diff
@@ -22,13 +22,15 @@ repos:
       - id: isort
 
   - repo: https://github.com/asottile/pyupgrade
-    rev: "v2.34.0"
+    rev: "v2.37.1"
     hooks:
       - id: pyupgrade
+        name: pyupgrade
         args: ["--py310-plus"]
+        exclude: docs
```

Comment:
This is excluded because pyupgrade was malfunctioning with our sphinx-js / python 3.10 compatibility codes.